src/test/java/org/icgc/argo/program_service/services/UpdateProgramTest.java

```diff
@@ -1,287 +0,0 @@
-package org.icgc.argo.program_service.services;
-
-import lombok.val;
-import org.icgc.argo.program_service.model.entity.CancerEntity;
-import org.icgc.argo.program_service.model.entity.PrimarySiteEntity;
-import org.icgc.argo.program_service.model.exceptions.NotFoundException;
-import org.icgc.argo.program_service.repositories.*;
-import org.icgc.argo.program_service.repositories.query.*;
-import org.icgc.argo.program_service.utils.EntityGenerator;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.junit4.SpringRunner;
-import org.springframework.transaction.annotation.Transactional;
-import java.util.Collections;
-import java.util.List;
-import static java.util.stream.Collectors.toList;
-import static org.junit.jupiter.api.Assertions.*;
-
-@SpringBootTest
-@RunWith(SpringRunner.class)
-@Transactional
-public class UpdateProgramTest {
```

Comment:
even though this test was a false positive, we still need to test the update functionality outside of a persistence context (i.e grpc integtration test).

Reply:
This will need to be part of a larger refactor.